framework/src/modules/chain/synchronizer/utils.js

```diff
@@ -75,11 +75,23 @@ const clearBlocksTempTable = storageModule =>
 const deleteBlocksAfterHeight = async (
 	processorModule,
 	blocksModule,
+	logger,
```

Comment:
it seems bit weird to have logger in the function

Reply:
Agreed, we can refactor this functions and move them to base synchronizer in another issue to not delay this fix

Reply:
Issue: #4541

Reply:
@limiaspasdaniel #4541